#! /bin/sh /usr/share/dpatch/dpatch-run
## link_system_libs.dpatch by Mirco Bauer <meebey@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Link against system libraries in preference to versions bundled in the
## DP: f-spot tarball.

@DPATCH@
diff -urNad f-spot~/Makefile.am f-spot/Makefile.am
--- f-spot~/Makefile.am	2008-04-16 22:08:41.000000000 +0100
+++ f-spot/Makefile.am	2008-04-16 22:08:42.000000000 +0100
@@ -11,7 +11,6 @@
 	libgphoto2-sharp \
 	semweb		\
 	Tao		\
-	mono-addins	\
 	tools		\
 	po		\
 	src		\
@@ -30,7 +29,6 @@
 	libgphoto2-sharp \
 	semweb		\
 	Tao		\
-	mono-addins	\
 	tools		\
 	po		\
 	src		\
diff -urNad f-spot~/Makefile.include f-spot/Makefile.include
--- f-spot~/Makefile.include	2008-04-16 22:07:51.000000000 +0100
+++ f-spot/Makefile.include	2008-04-16 22:08:42.000000000 +0100
@@ -27,10 +27,10 @@
 LINK_KEYRING = -r:$(DIR_KEYRING)/gnome-keyring-sharp.dll
 LINK_GLITZ = -r:$(DIR_GLITZ)/NDesk.Glitz.dll 
 LINK_GPHOTO2 = -r:$(DIR_GPHOTO2)/libgphoto2-sharp.dll
-LINK_MONO_ADDINS =					\
-	-r:$(DIR_ADDINS_ADDINS)/Mono.Addins.dll		\
-	-r:$(DIR_ADDINS_SETUP)/Mono.Addins.Setup.dll	\
-	-r:$(DIR_ADDINS_GUI)/Mono.Addins.Gui.dll
+LINK_MONO_ADDINS =			\
+	-pkg:mono-addins		\
+	-pkg:mono-addins-setup		\
+	-pkg:mono-addins-gui
 LINK_SEMWEB = -r:$(DIR_SEMWEB)/SemWeb.dll
 LINK_TAO = 								\
 	-r:$(DIR_TAO_OPENGL)/Tao.OpenGl.dll				\
diff -urNad f-spot~/configure.in f-spot/configure.in
--- f-spot~/configure.in	2008-04-16 22:08:41.000000000 +0100
+++ f-spot/configure.in	2008-04-16 22:09:09.000000000 +0100
@@ -283,10 +283,6 @@
 libjpegtran/Makefile
 libfspot/Makefile
 libgphoto2-sharp/Makefile
-mono-addins/Makefile
-mono-addins/Mono.Addins/Makefile
-mono-addins/Mono.Addins.Gui/Makefile
-mono-addins/Mono.Addins.Setup/Makefile
 semweb/Makefile
 tools/Makefile
 po/Makefile.in
